src/components/PopUpForm/PopUpForm.tsx

```diff
@@ -0,0 +1,274 @@
+import {
+  Form,
+  FormLayout,
+  TextField,
+  Button,
+  Card,
+  ActionList,
+  Popover,
+  Stack,
+  TextStyle,
+  Banner,
+} from "@shopify/polaris";
+import React, { useState } from "react";
+import { postalCodeIsValid } from "../../utils";
+
+export function PopUpForm() {
+  const [shouldShowInvalidName, setShouldShowInvalidName] = useState(false);
+  const [shouldShowInvalidDate, setShouldShowInvalidDate] = useState(false);
+  const [shouldShowInvalidAddress, setShouldShowInvalidAddress] = useState(
+    false,
+  );
+  const [shouldShowInvalidCity, setShouldShowInvalidCity] = useState(false);
+  const [shouldShowInvalidProvince, setShouldShowInvalidProvince] = useState(
+    false,
+  );
+  const [shouldShowInvalidPostal, setShouldShowInvalidPostal] = useState(false);
+  const [shouldShowInvalidURL, setShouldShowInvalidURL] = useState(false);
+  const [name, setName] = useState("");
+  const [date, setDate] = useState("");
+  const [address, setAddress] = useState("");
+  const [city, setCity] = useState("");
+  const [province, setProvince] = useState("");
+  const [postalCode, setPostalCode] = useState("");
+  const [phoneNumber, setPhoneNumber] = useState("");
+  const [website, setWebsite] = useState("");
+  const [numAvailable, setNumAvailable] = useState("");
+  const [vaccineTypeString, setVaccineTypeString] = useState(
+    "Select Vaccine Type",
+  );
+  /* eslint-disable  @typescript-eslint/no-unused-vars */
+  const [vaccineType, setVaccineType] = useState(1);
+  /* eslint-enable  @typescript-eslint/no-unused-vars */
+  const [active, setActive] = useState(false);
```

Comment:
I'd suggest `isPopupActive` and `setIsPopupActive` to identify this as a boolean and what it is activating.